extensions/adobe/experience/workfront/categoryparameter.schema.json

```diff
@@ -0,0 +1,62 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/workfront/categoryparameter",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Work Category Attributes",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": [
+    "https://ns.adobe.com/experience/workfront/object",
+    "https://ns.adobe.com/experience/workfront/changeevent"
+  ],
+  "description": "Category Parameter related fields",
+  "definitions": {
+    "categoryparameter": {
+      "properties": {
+        "workfront:categoryparameter": {
+          "type": "object",
+          "title": "Category Parameter Attributes",
+          "properties": {
+            "workfront:categoryParameterExpressionID": {
+              "title": "Category Parameter Expression ID",
+              "description": "Expression ID of category parameter",
+              "type": "string"
+            },
+            "workfront:parameterID": {
+              "title": "Parameter ID",
+              "description": "Parameter ID of category parameter object",
+              "type": "string"
+            },
+            "workfront:parameterGroupID": {
+              "title": "Parameter Group ID",
+              "description": "Group ID to which the category parameter belongs to",
+              "type": "string"
+            },
+            "workfront:securityLevel": {
+              "title": "Security Level",
+              "description": "Security level the category parameter belongs to",
+              "type": "string"
+            },
+            "workfront:viewSecurityLevel": {
+              "title": "View Security Level",
+              "description": "Security level view",
+              "type": "string"
+            }
+          }
+        }
+      }
+    }
+  },
+  "allOf": [
+    {
+      "$ref": "#/definitions/wfobject"
```

Comment:
This needs to match the local definition

Reply:
this is updated